contracts/ERC677Receiver.sol

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.6.10;
+
+interface ERC677Receiver {
+    function onTokenTransfer(address _sender, uint _value, bytes memory _data) external;
```

Comment:
Please remove the leading underscore from parameters, it adds unnecessary noise, in my opinion.